custom_components/resmed_myair/models.py

```
@@ -0,0 +1,119 @@
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from dataclasses import dataclass
+from datetime import date
+from typing import Any, Self
+
+from homeassistant.util import dt as dt_util
+
+
+def _to_usage_minutes(raw_usage: Any) -> int | None:
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `raw_usage`

(ANN401)